packages/backend/src/libp2p/connectionsManager.ts

```diff
@@ -129,6 +132,7 @@ export class ConnectionsManager extends EventEmitter {
     this.socketIOPort = socketIOPort
     this.quietDir = this.options.env?.appDataPath || QUIET_DIR_PATH
     this.connectedPeers = new Map()
+    this.communityDataPath = `${this.options.env.appDataPath}/communityData.json`
```

Comment:
* Do we always have appDataPath in options.env? If not then maybe let's use this.quietDir
* I'd use path.join here (Windows paths compatibiliity)

Reply:
That's right